unit_tests/test_tlslite_keyexchange.py

```diff
@@ -1707,3 +1707,22 @@ def test___init___with_rfc7919_group(self):
 
         self.assertEqual(kex.generator, 2)
         self.assertEqual(kex.prime, RFC7919_GROUPS[0][1])
+
+    def test_calc_public_value(self):
+        kex = FFDHKeyExchange(GroupName.ffdhe2048, (3, 4))
+
+        private = 2
+        public = kex.calc_public_value(private)
+        # verify that numbers are zero-padded
```

Comment:
Shouldn't you also verify that their byte length matches the prime length?

Reply:
it verifies that the returned value matches exactly, so the below assertEqual verifies not only that the value matches the one provided but that it's not a bit different